pkg/cortex/modules.go

```diff
@@ -78,7 +78,7 @@ func (m moduleName) String() string {
 	case Configs:
 		return "configs"
 	case AlertManager:
-		return "alertmanager"
+		return "alert-manager"
```

Comment:
Why are you hyphenating here?

Reply:
Please see my comment in the PR submit

Reply:
Actually I believe you are right, Alertmanager is just one word. I was confused due to the variable name with an uppercased "Manager".